packages/api3-voting/test/voting.js

```diff
@@ -462,9 +462,9 @@ contract('API3 Voting App', ([root, holder1, holder2, holder20, holder29, holder
       const voteId = createdVoteId(await voting.newTokenAndVote(holder2, 1, 'metadata'));
 
       const { snapshotBlock, votingPower } = await voting.getVote(voteId);
-      assertBn(snapshotBlock, await web3.eth.getBlockNumber() - 1, 'snapshot block should be correct');
+      assertBn(snapshotBlock, await web3.eth.getBlockNumber(), 'snapshot block should be correct');
       assertBn(votingPower, await token.totalSupplyAt(snapshotBlock), 'voting power should match snapshot supply');
-      assertBn(votingPower, 2, 'voting power should be correct')
```

Comment:
It looks like getting the snapshot in one block past protects against tokens being minted in a block and being used to vote in the same block. How is that a problem though?